operator/go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/pulumi/pulumi-kubernetes-operator/operator
 go 1.22.5
 
 require (
-	github.com/fluxcd/pkg/http/fetch v0.11.0
```

Comment:
Odd to see this package be removed.

Reply:
It's only needed by the agent. Overhead from juggling 2 mod files ;)

```
~/src/pulumi-kubernetes-operator/operator
❯ go mod why github.com/fluxcd/pkg/http/fetch
# github.com/fluxcd/pkg/http/fetch
(main module does not need package github.com/fluxcd/pkg/http/fetch)

~/src/pulumi-kubernetes-operator/agent
❯ go mod why github.com/fluxcd/pkg/http/fetch
# github.com/fluxcd/pkg/http/fetch
github.com/pulumi/pulumi-kubernetes-operator/agent/cmd
github.com/fluxcd/pkg/http/fetch
```